clients/web/src/templates/body/searchResults.pug

```diff
@@ -1,16 +1,15 @@
 .g-search-results-header-container
   .g-search-results-title-container
     i.g-search-results-title-element.icon-search
-    if length === 1
-      span.g-search-results-title.g-search-results-title-element Result : #{query}
-    else
-      span.g-search-results-title.g-search-results-title-element Results : #{query}
+    span.g-search-results-title.g-search-results-title-element Results : #{query}
```

Comment:
So we don't need to make the difference for only one result found ?

Reply:
I don't think it matters in this context, since it's a header for any number of items below, as opposed to part of a direct label (like the user count).